src/Microsoft.Data.SqlClient/src/Microsoft/Data/SqlClient/SqlInternalConnectionSmi.stub.cs

```diff
@@ -0,0 +1,16 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+#if NETFRAMEWORK
+
+namespace Microsoft.Data.SqlClient
+{
+    // DO NOT USE THIS FILE IN ANY PROJECT!
+    // This is a temporary stub to enable migrating DbConnectionInternal to the common project.
+    internal class SqlInternalConnectionSmi
+    {
+    }
```

Comment:
For awareness: I'll de-stub this once the context connections PR is merged.

Reply:
Works for me! I'm adding this for now just so I don't get red squigglies when working in the common project. (at least for the files in this PR - I know a lot of other ones in the project are missing dependencies in the common project)